tests/performance-tests/atala-performance-tests-k6/README.md

```diff
@@ -29,3 +32,27 @@ Once that is done, we can run our script the same way we usually do, for instanc
 ```bash
 $ k6 run dist/connection-flow-test.js
 ```
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+# Debugging Tests
+=======
+## Debugging Tests
+>>>>>>> 7aa9b4c2 (feat: upgrade ZIO http client to improve performance (#850))
+=======
+## Debugging Tests
+>>>>>>> 7aa9b4c2 (feat: upgrade ZIO http client to improve performance (#850))
+
+k6 can be configured to log the HTTP request and responses that it makes during test execution. This is useful to debug errors that happen in tests when logs or k6 output does not contain the reason for a failure.
+
+For example, if many requests result in 503s due to HTTP timeouts, there aren't many logs available to show when and why this happened.
+
+To enable k6 to output requests and responses - add the `--http-debug` flag to the k6 test execution command
```

Comment:
To enable k6 to output requests and responses - add the `--http-debug` flag to the k6 test execution command.